docs_outdoornav_user_manual/features/points_of_interest.mdx

```diff
@@ -0,0 +1,59 @@
+---
+title: Points of Interest
+sidebar_label: Points of Interest
+sidebar_position: 6
+toc_min_heading_level: 2
+toc_max_heading_level: 4
+---
+
+
+As of version 0.13.0, the OutdoorNav software contains the ability for users to set
+up Points of Interest (POI) on their map. POIs are intended to be static locations,
+landmarks, objects of interest, etc... that a user is able to either navigate to or inspect.

```

Comment:
Not sure but I think here it would just be:
objects of interest, etc. 
No need for the ellipses.